### ISSUE TITLE
The "react-scripts: not found" error occures when running frontend react server

### ISSUE THREAD

**Issue by ani-hovhannisyan** [ani-hovhannisyan/kanji-visualization#13]
The following error occurs when trying to run frontend environment in local PC without docker (react server):
**$ yarn start**
```
yarn run v1.22.17
$ react-scripts start
/bin/sh: 1: react-scripts: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

**Comment by ani-hovhannisyan** [comment]
My local machine is Ubuntu 20.04.3 LTS, x64.
The "--ignore-engines" argument for "yarn install" in frontend folder solved this error.
Should we run like this?
```
cd frontend
yarn install --ignore-enginesl
```

**Comment by wowry** [comment]
Can you please tell me what version of node.js you are using?
```
node -v
```

**Comment by ani-hovhannisyan** [comment]
NodeJS is v10.19.0.

**Comment by wowry** [comment]
@ani-hovhannisyan 
Thanks!

Could you please try to get the latest version (v16) of Node.js?
https://joshtronic.com/2021/05/09/how-to-install-nodejs-16-on-ubuntu-2004-lts/

Then, I guess you can `yarn install` without `--ignore-engines`.

**Comment by ani-hovhannisyan** [comment]
Ok thanks. Could you also update the lowest version of all necessary dependencies in Readme, so we won't try with oldest ones?

**Comment by wowry** [comment]
OK, I'll add system requirements to README.

**Comment by ani-hovhannisyan** [comment]
To keep the track these are my local versions.
pip v20.0.2, python v3.8.10, NodeJS v10.19.0, Yarn, v1.22.17.

**Comment by ani-hovhannisyan** [comment]
Hura! Updating NodeJS version from "v10.19.0" to "v16.13.2" fixed both #13 and #18 bugs.
I'm opening new #19 ongoing task to update Readme periodically during Milestones until we reach completed development environment preparation guidance #19

**Comment by wowry** [comment]
We did it, thanks!
